docs/usage.rst

```diff
@@ -95,19 +95,23 @@ To validate a RAML file with Python:
 
    >>> from ramlfications import validate
    >>> RAML_FILE = "/path/to/my-api.raml"
-   >>> validate(RAML_FILE)
-   >>>
+   >>> errors = validate(RAML_FILE)
+   >>> if not errors:
+   >>>     print("Success!")
 
 .. code-block:: python
 
    >>> from ramlfications import validate
    >>> RAML_FILE = "/path/to/invalid/no-title.raml"
-   >>> validate(RAML_FILE)
-   InvalidRootNodeError: RAML File does not define an API title.
+   >>> errors = validate(RAML_FILE)
```

Comment:
More commenting on the implemented behavior rather than the docs:

This breaks the behavior of the current API of the package.

Currently, the `validate` function returns nothing if the file is valid, and the (first) error if it's not:

```pyc
>>> RAML_FILE = "invalid.raml"
>>> validate(RAML_FILE)
InvalidParameterError: '422' not a valid HTTP response code.
>>> RAML_FILE = "valid.raml"
>>> validate(RAML_FILE)
>>>
```

However, with this, it's now:

```pyc
>>> RAML_FILE = "invalid.raml"
>>> validate(RAML_FILE)
[ramlfications.errors.InvalidParameterError("'422' not a valid HTTP response code.")]
>>> RAML_FILE = "valid.raml"
>>> validate(RAML_FILE)
[]
>>>
```

The proposed implementation returns a list in either case, the original returns nothing if successful, and an error message if invalid.   I'd like to at least keep the "return nothing if valid" behavior.  To be honest, not sure what I'd like to have for invalid. But when originally implemented, I wanted to be able to do something like this:

```python
try:
    validate(RAML_FILE)
except Exception:
    raise SomeError
```

So with the proposed implementation, the `try` will succeed and not hit the `except` clause.